packages/calcite-components/src/components/tip-manager/tip-manager.scss

```diff
@@ -1,31 +1,39 @@
+@import "../../assets/styles/header";
+
 /**
- * CSS Custom Properties
- *
- * These properties can be overridden using the component's tag as selector.
- *
- * @prop --calcite-tip-manager-action-background-color: defines the background color of an action sub-component inside the component.
- * @prop --calcite-tip-manager-action-text-color: defines the text color of an action sub-component inside the component.
- * @prop --calcite-tip-manager-action-background-color-hover: defines the background color of an action sub-component when hovered or focused inside the component.
- * @prop --calcite-tip-manager-action-text-color-hover: defines the text color of an action sub-component when hovered or focused inside the component.
- * @prop --calcite-tip-manager-action-background-color-active: defines the background color of an action sub-component when active inside the component.
- * @prop --calcite-tip-manager-action-text-color-active: defines the text color of an action sub-component when active inside the component.
- * @prop --calcite-tip-manager-height: The maximum height of the component.
- * @prop --calcite-tip-max-width: The maximum width of a slotted `calcite-tip` within the component.
- *
- */
+* CSS Custom Properties
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-tip-max-width: [Deprecated] No longer necessary, as we don't use mapped tokens for slotted components. Specifies the maximum width of a slotted`calcite-tip` within the component.
```

Comment:
Since this was documented in the doc-site, we need to point users to the updated token name.